pkg/storages/memory/folder.go

```diff
@@ -19,6 +19,7 @@ type Folder struct {
 }
 
 func NewFolder(path string, storage *Storage) *Folder {
+	path = strings.TrimPrefix(path, "/")
```

Comment:
Why do we trim it?

Reply:
This memory folder in its implementation of `ListFolder` func decides that a file is stored inside a folder if their prefixes are the same. However, file paths never start with "/", but the folder one do. So it doesn't assign any files to such folder. The same problem may be with other funcs too. 

I suppose it would be great to remove the ambigutiy between "/a/b/c" and "a/b/c" paths (as well as between "/" and "") in all our `storage.Folder` implimentations, because there may be quite a few bugs related to that. But this is too much to do it right here. So I fixed only the problem I found with my tests.